python/ray/dashboard/subprocesses/routes.py

```diff
@@ -88,6 +88,7 @@ async def parent_side_handler(
             # Used in bound_routes().
             parent_side_handler.__route_method__ = method
             parent_side_handler.__route_path__ = path
+            parent_side_handler.__handler__ = handler
```

Comment:
should we do `parent_side_handler.__name__ = handler.__name__` instead?

Reply:
Updated